libpod/container_api.go

```diff
@@ -821,3 +821,12 @@ func (c *Container) Restore(ctx context.Context, options ContainerCheckpointOpti
 	defer c.newContainerEvent(events.Restore)
 	return c.restore(ctx, options)
 }
+
+// AutoRemove indicates whether the container will be removed after it is executed
+func (c *Container) AutoRemove() bool {
+	spec := c.Spec()
```

Comment:
`c.Spec()` is a VERY expensive call - please use `c.config.Spec` instead

Reply:
Fixed